From 0fb0b6a6f19e3069f9e326253b05d3e0b35d27c5 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 19 Nov 2019 22:51:59 +0100
Subject: [PATCH 1/1] arm: fix -march for ARM11

-march v5 is invalid for GCC 9.2.1. ARM11 is an armv6 implementation. So
change the architecture flag for the compiler to armv6.
Cf. https://gcc.gnu.org/onlinedocs/gcc/ARM-Options.html

Suggested-by: Fabio Estevam <festevam@gmail.com>
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 arch/arm/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/Makefile b/arch/arm/Makefile
index 5384981c17..60af7e3199 100644
--- a/arch/arm/Makefile
+++ b/arch/arm/Makefile
@@ -12,8 +12,8 @@ arch-$(CONFIG_CPU_ARM926EJS)	=-march=armv5te
 arch-$(CONFIG_CPU_ARM946ES)	=-march=armv5te
 arch-$(CONFIG_CPU_SA1100)	=-march=armv4
 arch-$(CONFIG_CPU_PXA)		=
-arch-$(CONFIG_CPU_ARM1136)	=-march=armv5
-arch-$(CONFIG_CPU_ARM1176)	=-march=armv5t
+arch-$(CONFIG_CPU_ARM1136)	=-march=armv6
+arch-$(CONFIG_CPU_ARM1176)	=-march=armv6
 arch-$(CONFIG_CPU_V7A)		=$(call cc-option, -march=armv7-a, \
 				 $(call cc-option, -march=armv7))
 arch-$(CONFIG_CPU_V7M)		=-march=armv7-m
-- 
2.24.0

